e2e/test/scenarios/models/reproductions.cy.spec.js

```diff
@@ -530,93 +530,97 @@ describe.skip("issue 22519", () => {
   });
 });
 
-describe("filtering based on the remapped column name should result in a correct query (metabase#22715)", {tags: "@flaky"}, () => {
-  function mapColumnTo({ table, column } = {}) {
-    cy.findByText("Database column this maps to")
-      .parent()
-      .contains("None")
-      .click();
+describe(
```

Comment:
This is unrelated to the PR, but prettier was failing on this file.

Reply:
FYI [fix](https://github.com/metabase/metabase/pull/57119) is in `master`